src/spritefeat.asm

```diff
@@ -0,0 +1,330 @@
+;
+; Features using sprites to draw information
+;
+
+
+; Hook Gamemode 7 to upload tiles after loading if needed
+org $828B26
+    JSL update_sprite_tiles_loading
+
+; Hook sprite uploads in main game loop to get a chance to upload sprites first
+; Runs in Gamemode 8
+org $828B98
+    JSL update_sprite_features
+
+
+
+;
+; Add hitbox graphic to free sprite VRAM slot
+;
+org $9ADAE0
+db $FF, $FF, $80, $80, $80, $80, $80, $80, $80, $80, $80, $80, $80, $80, $80, $80
+db $FF, $FF, $80, $80, $80, $80, $80, $80, $80, $80, $80, $80, $80, $80, $80, $80 
+
+
+org $F0E000
+print pc, " spritefeat start"
+; This runs every frame before any other sprites are drawn, this is needed so we can get priority over everything else
+update_sprite_features:
+    PHA : PHX : PHY : PHP
+    
+    ; Draw OOB viewer if activated
+    JSR update_sprite_oob
+
+    ; Draw Samus hitbox if activated
+    JSR update_sprite_hitbox
+
+    PLP : PLY : PLX : PLA
+    JSL $A0884D    
```

Comment:
if !FEATURE_PAL
    JSL $A0885D
else
   JSL $A0884D
endif

Didn't find any other PAL conflicts.

Reply:
Oh right, PAL is a thing we do now. I completely forgot about that :)
Thanks for noticing, I'll add that in.